frontend/public/components/image-stream.tsx

```diff
@@ -190,13 +190,10 @@ export const ExampleDockerCommandPopover: React.FC<ImageStreamManipulationHelpPr
         </div>
       }
     >
-      <button
-        className="btn btn-link btn-link--no-btn-default-values hidden-sm hidden-xs"
```

Comment:
Do you have a follow on task to remove the `btn-link--` classes?  Or are you hoping to get all instances in this PR and also remove them?

Reply:
Yes, the plan is to remove any `.btn-link*` with a PR where all instances have been removed.